### ISSUE TITLE
Fix dropdown with multiple selection

### ISSUE THREAD

**Issue by LiliumCandidum** [mia-platform/design-system#664]
## Fix dropdown with multiple selection
The dropdown component with multiple selection needs some improvements:
- it should appear a check icon when an item is selected
- it shoudn't close when an item is selected/deselected
- the color "on hover" of a selected item should be lighter

## A snippet of code for replicating the issue or showing the proposal usage if applicable
See storybook [multiple highlight selection](https://mia-platform.github.io/design-system/?path=/docs/components-dropdown--docs#multiple-highlight-selection)

## The expected result for your bug
See [Dropdown documentation](https://www.figma.com/design/0ar6alIEDe8iYCb8kU7Rxd/%5BDS%5D-Console-Mia-Platform?node-id=519-8488)

**Comment by fredmaggiowski** [comment]
> it shoudn't close when an item is selected/deselected

regarding this I worry that it's the default behaviour controlled by antd, I'm not 100% confident it will be something feasible tbh

**Comment by LiliumCandidum** [comment]
> regarding this I worry that it's the default behaviour controlled by antd, I'm not 100% confident it will be something feasible tbh

I think that with `open` and `onOpenChange` props of the antd dropdown it's possible to control the open/close behaviour and implement it

**Comment by fredmaggiowski** [comment]
> I think that with open and onOpenChange props of the antd dropdown it's possible to control the open/close behaviour and implement it

Yes it is, but messing with the `open` prop will lead to lots of troubles: if you start controlling the `open` prop you will have to manually control it for all the available triggers and `onOpenChange` will become useless, as you already know when the "open" state changed, since you are given by antd full control.
